Write Automate section (#35)
directus/docs: content/7.automate/2.flows.md

- **Status** — Sets the flow to active or inactive.
- **Icon** — Adds an icon to help quickly identify the flow.
- **Description** — Sets a brief verbal description of the flow.
- **Color** — Sets a color to help identify the flow.
- **Activity and Logs Tracking** — Selects whether the flow should be tracked in both the [activity log](auth/accountability) and the flow's logs, only in the activity log, or neither.

## Logs

<!-- TODO: Image -->

Accessible from the sidebar, logs store information for each flow execution. Each log will display information from
triggers as well as each operation in the flow.

Logs are not a 1:1 mapping to the data chain. Each trigger and operation gets its own dropdown, which stores its
relevant data. Here's what you'll get from each of these:

### Trigger

- **Options** — The values you input when you configured the trigger.
- **Payload** — Displays the data appended under `$trigger`, which includes the data passed into the flow itself.
- **Accountability** — Displays data appended under `$accountability`, which is a breakdown of the permissions and request information of the user that triggered that event.

> Note that `$accountability` is not nested under the `$trigger` key. However, it is listed under the Trigger in the Log

Reviewer: ```suggestion
Note that `$accountability` is not nested under the `$trigger` key. However, it is listed under the trigger in the log
```

## Fields

- **Name (required)** — The displayed name of the Flow.
- **Status** — Sets the flow to active or inactive.
- **Icon** — Adds an icon to help quickly identify the flow.
- **Description** — Sets a brief verbal description of the flow.
- **Color** — Sets a color to help identify the flow.
- **Activity and Logs Tracking** — Selects whether the flow should be tracked in both the [activity log](auth/accountability) and the flow's logs, only in the activity log, or neither.

## Logs

<!-- TODO: Image -->

Accessible from the sidebar, logs store information for each flow execution. Each log will display information from
triggers as well as each operation in the flow.

Logs are not a 1:1 mapping to the data chain. Each trigger and operation gets its own dropdown, which stores its
relevant data. Here's what you'll get from each of these:

### Trigger

- **Options** — The values you input when you configured the trigger.
- **Payload** — Displays the data appended under `$trigger`, including the event triggered, the key of the event, the collection affected, and more. The payload itself contains the affected data.

> - **Accountability** — Displays data appended under `$accountability`, which is a breakdown of the permissions and request information of the user that triggered that event..

Reviewer: ```suggestion
- **Accountability** — Displays data appended under `$accountability`, which is a breakdown of the permissions and request information of the user that triggered that event.
```

---

> description: Learn to use Flows in Directus Automate.

Reviewer: ```suggestion
description: Flows enable custom, event-driven data processing and task automation within Directus.
```

---

# Flows

> Flows enable custom, event-driven data processing and task automation within Directus. Each flow is composed of one trigger, a series of operations and a data chain.

Reviewer: ```suggestion
Flows enable custom, event-driven data processing and task automation within Directus. Each flow is composed of one trigger, a series of operations, and a data chain that is passed between each step.
```

## Fields

- **Name** — The displayed name of the Flow.
- **Status** — Sets the flow to active or inactive.
- **Icon** — Adds an icon to help quickly identify the flow.
- **Description** — Sets a brief verbal description of the flow.
- **Color** — Sets a color to help identify the flow.

> - **Activity and Logs Tracking** — Lets you **Track Activity and Logs**, **Activity**, or **Neither**.

Reviewer: Explain this option more

::callout{type="info"}

"Name" is the only required field.

> ::

Reviewer: I think this is a lot for just marking one as required - instead maybe add `(required)` above?

## Logs

<!-- TODO: Image -->

Accessible from the sidebar, logs store information for each flow execution. Each log will display information from

> triggers as well as each operation in the flow. To access a flow's logs, follow these steps.

Reviewer: ```suggestion
Accessible from the sidebar, logs store information for each flow execution. Each log will display information from
triggers as well as each operation in the flow.
```

Logs are not a 1:1 mapping to the data chain. Each trigger and operation gets its own dropdown, which stores its
relevant data. Here's what you'll get from each of these:

### Trigger

- **Options** — The values you input when you configured the trigger.

> - **Payload** — Displays the data appended under `$trigger`.

Reviewer: Explain what the payload is

Author: data being passed into the flow

> - **Accountability** — Displays data appended under `$accountability`.

Reviewer: Explain what accountability is

Note that `$accountability` is not nested under the `$trigger` key. However, it is listed under the Trigger in the Log
because `$accountability` is generated by the trigger.

### `<OperationKey>`

- **Options** — The values you input when you configured the operation.\
  _(These configuration options are not stored on the data chain)_.
- **Payload** — Displays the data appended under this `<operationKey>`.

> 

Reviewer: Is there not also another outcome/result one too?

::callout{type="warning" title="Logs are stored in the database"}

Keep in mind that if you've configured a flow to track logs, all this information is stored in the database. You may

> need to periodically delete this data.

Reviewer: Can you make this a bit more succinct? It's quite chatty like the previous docs

::

::callout{type="info" title="Where is `$last`?"}

You may notice `$last` is not in the Logs. Remember, `$last` constantly updates to store the data of the most recently
executed operation. The log shows the results of the entire flow. Therefore `$last` would simply be the very last

> operation in the flow.

Reviewer: Same here - too verbose and chatty for a manual
